tests/SbrpTests/DummyAttributeTypeProvider.cs

```diff
@@ -6,25 +6,25 @@
 using System.Reflection.Metadata;
 using System.Reflection.Metadata.Ecma335;
 
-namespace ValidateSbrpTests;
+namespace SbrpTests;
 
 internal class DummyAttributeTypeProvider : ICustomAttributeTypeProvider<Type>
 {
     public static readonly DummyAttributeTypeProvider Instance = new();
 
-    public Type GetPrimitiveType(PrimitiveTypeCode typeCode) => default(Type);
+    public Type GetPrimitiveType(PrimitiveTypeCode typeCode) => default(Type)!;
```

Comment:
The `!` portion of this doesn't seem correct. `Type` is a `class`, so `default(Type)` will be `null`. The return type of the method is `Type`, which is non-nullable. So yes, adding `!` takes care of the compiler error but it doesn't make it correct since it will be null at runtime and break the nullability contract. The correct thing to do would be to change it from `ICustomAttributeTypeProvider<Type>` to `ICustomAttributeTypeProvider<Type?>`, and all the return types to be `Type?`. That will allow `null` or `default(Type)` to be returned.

Reply:
That makes sense! I was under the assumption that it had to be `ICustomAttributeTypeProvider<Type>`. Changed to `ICustomAttributeTypeProvider<Type?>` and made related changes in https://github.com/dotnet/source-build-reference-packages/pull/817/commits/8ab54ea0adfc0dd81813d7d749c06f7eb416dbce.